### PR TITLE
Swagger closure for Azure Migration Discovery service

### PR DESCRIPTION
**Pull request by pajind**
Addition of new resources "serversite", "importsite", "vmwaresite", "hypervsite"

This PR is raised to add new resource "serversite", "importsite".  Resources "vmwaresite" and "hypervsite" were already added earlier but its swagger was not generated so its added now.

### CODE REVIEW

File: specification/migrate/resource-manager/Microsoft.OffAzure/2018-05-01-preview/migrate.json [hunk 1]
```diff
@@ -0,0 +1,6180 @@
...
+        ],
+        "produces": [
+          "application/json",
+          "text/json",
+          "application/xml",
+          "text/xml"
+        ],
+        "parameters": [
+          {
+
```
Comment (nschonni) [inline specification/migrate/resource-manager/Microsoft.OffAzure/2018-05-01-preview/migrate.json]:
```suggestion
        "staticIpAddress": {
```

**Comment by pajind** [comment]
Hi, We don't want to publish this swagger publicly. It is just for adding new resources in this namespace. Tell us what needs to be done further.

**Comment by sanjaiganesh** [comment]
@pajind There is a PR in private repo already on this, which we looked at.. I suppose you would like to drop this PR, as you requested to review the private repo one. Can you confirm ?
https://github.com/Azure/azure-rest-api-specs-pr/pull/769/files

**Comment by pajind** [comment]
It is raised in private repo so closing it.

**Comment by pajind** [comment]
Dropped the one in public repo : https://github.com/Azure/azure-rest-api-specs/pull/6004 .

-pallavi

**Pull request closed without merging**